spectacles.gemspec

```diff
@@ -22,12 +22,13 @@ Gem::Specification.new do |gem|
   # Dependencies
   #
   gem.required_ruby_version = ">= 2.2.0"
-  gem.add_dependency "activerecord", ">= 3.2.0", "< 6.0"
-  gem.add_dependency "activesupport", ">= 3.2.0", "< 6.0"
+  gem.add_dependency "activerecord", ">= 3.2.0", "~> 6.1.0"
```

Comment:
I haven't cut Rails version-specific versions of Spectacles so far because I haven't needed to. But it seems that it might be time to start dropping support for old Rails versions, and/or keeping the Spectacles versions aligned with Rails versions. Thoughts?

Reply:
I think we're fine to cut a Rails 6.0+ version as `ActiveRecord::Base.connection_handler` is a 6.0+ feature.